extensions/SharkPool/SoundCloud-API.js

```diff
@@ -0,0 +1,363 @@
+
+  let clientID = "8BBZpqUP1KSN4W6YB64xog2PX4Dw98b1";
+  const proxy = "https://corsproxy.io?";
```

Comment:
I shouldn't need to say this but an entire API extension relying on a cors proxy is pretty bad. If so much as the cors proxy or the api breaks, the extension will break, and **real projects** by **real people** will also break one day for no reason.